test/unit/routes/test-advanced-search.js

```diff
@@ -70,21 +117,45 @@ describe('routes/index', function () {
     var start = 0
     var length = 10
 
-    it('should respond with a 200 and pass query object to data layer', function () {
-      var searchCriteria = {
-        reference: 'A123456',
-        name: 'John Smith'
-      }
+    it('should respond with a 200 and call data method', function () {
       getClaimListForAdvancedSearch.resolves({claims: [RETURNED_CLAIM], total: {Count: 1}})
       return supertest(app)
-        .get(`/advanced-search-results?reference=${searchCriteria.reference}&name=${searchCriteria.name}&draw=${draw}&start=${start}&length=${length}`)
+        .get(`/advanced-search-results?&draw=${draw}&start=${start}&length=${length}`)
         .expect(200)
         .expect(function (response) {
           expect(isCaseworkerStub.calledOnce).to.be.true
-          expect(getClaimListForAdvancedSearch.calledWith(searchCriteria, start, length), 'expected data method to be called with parameters').to.be.true
```

Comment:
Why was this line removed?  Should still be checking if the getClaimListForAdvancedSearch function is called with an empty searchCriteria object